tmail-backend/apps/memory/pom.xml

```diff
@@ -179,6 +179,17 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>net.alchim31.maven</groupId>
+                <artifactId>scala-maven-plugin</artifactId>
```

Comment:
Sorry for bothering but things like guice modules do not need to be scala.

It adds build delay.

Is it possible to rewrite those in java and drop this plugin?